kubernetes/resource_kubernetes_pod.go

```diff
@@ -24,6 +24,15 @@ func resourceKubernetesPod() *schema.Resource {
 		Importer: &schema.ResourceImporter{
 			State: schema.ImportStatePassthrough,
 		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create:  schema.DefaultTimeout(5 * time.Minute),
+			Read:    schema.DefaultTimeout(5 * time.Minute),
+			Update:  schema.DefaultTimeout(5 * time.Minute),
+			Delete:  schema.DefaultTimeout(5 * time.Minute),
+			Default: schema.DefaultTimeout(5 * time.Minute),
+		},
```

Comment:
These need to be referenced in the code that would use them. 

See:

- https://github.com/terraform-providers/terraform-provider-aws/blob/d5d8511e754406d10ecd108bb892e2f3296814d3/aws/resource_aws_db_instance.go#L528-L541

- https://github.com/terraform-providers/terraform-provider-aws/blob/d5d8511e754406d10ecd108bb892e2f3296814d3/aws/resource_aws_rds_cluster_instance.go#L474-L490

Reply:
womp womp. Thank you for the links I was wondering how it worked.